apps/chatbot/src/modules/utils.py

```diff
@@ -1,25 +1,41 @@
+import os
 import boto3
 import logging
+from dotenv import load_dotenv
 
-def get_ssm_parameter(name: str, default: str | None = None) -> str | None:
+load_dotenv()
+
+
+GOOGLE_PARAM_NAME = os.getenv("GOOGLE_PARAM_NAME")
+AWS_ACCESS_KEY_ID = os.getenv("CHB_AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.getenv("CHB_AWS_SECRET_ACCESS_KEY")
+AWS_DEFAUL_REGION = os.getenv("GOOGLE_AND_REDIS_AWS_DEFAULT_REGION")
+
+
+def get_ssm_parameter(name: str = GOOGLE_PARAM_NAME, default: str | None = None) -> str | None:
```

Comment:
If you agree, I would pass the parameter `name` from the outside. Hence, I would consider removing the default value.
It can be used for other ssm parameters in the future